src/ui/pages/ManageWallets/WalletGroup/WalletGroup.tsx

```diff
@@ -33,13 +33,29 @@ import { WalletAvatar } from 'src/ui/components/WalletAvatar';
 import { InputDecorator } from 'src/ui/ui-kit/Input/InputDecorator';
 import { BackupInfoNote } from 'src/ui/components/BackupInfoNote';
 import {
+  ContainerType,
+  getContainerType,
   isHardwareContainer,
   isMnemonicContainer,
-  isPrivateKeyContainer,
   isSignerContainer,
 } from 'src/shared/types/validators';
 import { NeutralDecimals } from 'src/ui/ui-kit/NeutralDecimals';
 
+const strings = {
+  recoveryPhraseTitle: 'Recovery Phrase',
+  privateKeyTitle: 'Private Key',
+  removeWalletSubtitle: (containerType: ContainerType) =>
+    containerType === ContainerType.mnemonic
+      ? 'You can always import it again using your recovery phrase'
+      : containerType === ContainerType.privateKey
+      ? 'You can always import it again using your private key'
+      : containerType === ContainerType.hardware
+      ? 'You can import it again by connecting your hardware wallet'
+      : containerType === ContainerType.readonly
+      ? 'You can always add it back to your watch list'
+      : 'You can add it again on the Manage Wallets page',
```

Comment:
Do we need this fallback text?

Reply:
Good question. Do you think we should throw?

Reply:
Yes:)